packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2330,6 +2330,33 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     }
   }
 
+  /// Creates the selection overlay if it is null.
+  ///
+  /// Returns `true` if the overlay was created, and `false` if it already
+  /// existed.
```

Comment:
These triple slashes should be double because this method is private.

Reply:
Fixed!